packages/ckeditor5-ui/src/list/listview.ts

```diff
@@ -141,7 +142,7 @@ export default class ListView extends View<HTMLUListElement> implements Dropdown
 		for ( const item of this.items ) {
 			if ( item instanceof ListItemGroupView ) {
 				this._registerFocusableItemsGroup( item );
-			} else {
+			} else if ( !( item instanceof ListSeparatorView ) ) {
 				this._registerFocusableListItem( item );
 			}
```

Comment:
What if we add another type to this collection in the future? Then unfortunately this check would not be sufficient. So it might be better to explicitly check the needed class in `_registerFocusableListItem`, which is `ListItemView`. So instead of checking that it's not a separator, let's check if it's a list item.